packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -92,7 +90,5 @@ const ItemDisplay = ({
 ItemDisplay.propTypes = propTypes;
 ItemDisplay.defaultProps = defaultProps;
 
-ItemDisplay.Comment = ItemComment;
```

Comment:
This is a breaking change, we may want to spike on how much impact this will have. Kaiju, for example, has been generating all comments from `ItemDisplay.Comment`

Reply:
@bjankord @ryanthemanuel While searching in github.cerner, I found `ItemDisplay.Comment` to be used in a lot of places https://github.cerner.com/search?l=JSX&q=ItemDisplay.Comment&type=Code. Will this change be a `Major Version Bump`?

Reply:
We discussed it here and we'd like to actually move the clinical item display and comment to the clinical item view and then deprecate the item display.  We'll do this as part of the deprecation of item collection which is scheduled in the next month or so.